page/NursePage.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import {View} from 'react-native';
+import { Picker } from "@react-native-picker/picker";
```

Comment:
Standarize this imports. 

```suggestion
import { View } from 'react-native';
import { Picker } from "@react-native-picker/picker";
```
ó

```suggestion
import {View} from 'react-native';
import {Picker} from "@react-native-picker/picker";
```